src/libraries/grommet/components.js

```diff
@@ -2050,37 +2106,54 @@ export const components = {
   DataFilters: {
     component: DataFilters,
     name: 'DataFilters',
+    documentation: `${grommetWebsite}/datafilters`,
     defaultProps: {
-      drop: true,
+      clearFilters: true,
+      updateOn: 'submit',
     },
     properties: {
-      drop: true,
+      drop: false,
+      heading: '',
       layer: false,
+      clearFilters: false,
+      updateOn: ['change', 'submit'],
     },
   },
   DataFilter: {
     component: DataFilter,
     name: 'DataFilter',
-    properties: {},
+    documentation: `${grommetWebsite}/datafilter`,
+    properties: {
+      property: '',
```

Comment:
think we also need `options` and `range` 
https://v2.grommet.io/datafilter

Reply:
Let's leave this one to a separate PR since it's a little more complex to add.

Reply:
sounds good